### PR TITLE
build: add eslint with typescript

### PR DESCRIPTION
**Pull request by matmalkowski**

### CODE REVIEW

**Comment by matmalkowski** [comment]
@znut why you killed my branch?

**Comment by znut** [comment]
resolve conflicts please 🤣

**Comment by matmalkowski** [comment]
still WIP

File: .vscode/settings.json [hunk 1]
```diff
@@ -3,12 +3,24 @@
...
     "**/coverage": true,
     "**/lib": true,
     "**/node_modules": true
   },
-  "editor.defaultFormatter": "esbenp.prettier-vscode"
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "eslint.validate": [
+    {
+      "language": "typescript",
+      "autoFix": false
```
Comment (znut) [inline .vscode/settings.json]: we should auto fix? I think it's more convenient for me :D

File: packages/sandbox/src/i18nExamples/I18nExamples.tsx [hunk 2]
```diff
@@ -44,7 +44,7 @@ const I18nExamples: React.FC = () => {
       </p>
       <p>{objFormattedText}</p>
       <p>
-        <I18n k="example.invalid.or.missing.key">This default text is rendered because key doesn't exists</I18n>
+        <I18n k="example.invalid.or.missing.key">This default text is rendered because key doesn&#39;t exists</I18n>
```
Comment (znut) [inline packages/sandbox/src/i18nExamples/I18nExamples.tsx]: why we need to escape this `'` ?

File: packages/feature-toggle-jsx/package.json [hunk 3]
```diff
@@ -40,6 +40,7 @@
     "watch:code": "rollup -c -w",
     "watch:types": "yarn emit-declarations --watch",
     "typecheck": "tsc",
+    "lint": "eslint -c eslint.config.js --ext .ts ./src/",
```
Comment (znut) [inline packages/feature-toggle-jsx/package.json]: do we need it here? I see we lint already in the root work-space level

File: packages/sandbox/src/i18nExamples/I18nExamples.tsx (hunk 2 above)
Comment (matmalkowski) [reply packages/sandbox/src/i18nExamples/I18nExamples.tsx]: Linting rule says so ;)

File: .vscode/settings.json (hunk 1 above)
Comment (matmalkowski) [reply .vscode/settings.json]: Oh right, forgot to change this back

**Pull request merged**